scripts/update-versions.mjs

```
@@ -46,13 +46,13 @@ function updateVersionInCorePackageJsonFile(newVersion) {
 }
 
 function updateVersionInSourceFile(newVersion) {
-  const path = 'packages/core/src/Client.ts';
+  const path = 'packages/core/src/util/Constants.ts';
   console.info('Updating', path);
```

ESLint

[error] Unexpected console statement.

(no-console)